### PR TITLE
add prometheus integrations to multi_metadata.yaml

### PR DESCRIPTION
**Pull request by shyamvalsan**
Keywords still to be updated.

### CODE REVIEW

File: modules/prometheus/multi_metadata.yaml [hunk 1]
```diff
@@ -264,3 +264,7108 @@ modules:
...
+        - data-collection.networking-stack-and-network-interfaces
+      keywords:
+      - network monitoring
+      - network performance
+      - traffic analysis
+    overview:
+      <<: *overview: 
+      data_collection:
+        <<: *data_collection:
+        metrics_description: Monitor your A10-Networks metrics.
```
Comment (ilyam8) [inline modules/prometheus/multi_metadata.yaml:285]: I think we need to mention the exporter somewhere too. I think it needs to be in:

- `method_description` and

```
It collects metrics by periodically sending HTTP requests to [X exporter](LINK) and parsing the response.
```

- `setup.prerequisites.list` (Install X exporter)

**Review by sashwathn — APPROVED** [review]

**Comment by ilyam8** [comment]
- please make sentences single lines if using `|`. 

```yaml
        metrics_description: |
          Monitor GPSD (GPS daemon) metrics for efficient GPS
          data management and performance.
        method_description: |
          Metrics are gathered by periodically sending HTTP requests
          to [gpsd exporter](https://github.com/natesales/gpsd-exporter) and parsing
              the response.
```

=>

```
        metrics_description: |
          Monitor GPSD (GPS daemon) metrics for efficient GPS data management and performance.
        method_description: |
          Metrics are gathered by periodically sending HTTP requests to [gpsd exporter](https://github.com/natesales/gpsd-exporter) and parsing the response.
```

- can you please filter out things that are not on GitHub

As it is now we end up with

> Metrics are gathered by periodically sending HTTP requests https://www.webweaver.de/
> Install https://www.webweaver.de/ by following the instructions mentioned in the exporter README.

Please check https://www.webweaver.de/ and try to follow their instructions to install the "exporter".

I am very doubtful we need such "integrations".

**Comment by ilyam8** [comment]
I have no problems with 1000+ but the number is not the only goal, they at least shouldn't be super low quality/senseless. I don't say drop the rest, we need just to verify things.

**Comment by ilyam8** [comment]
@shyamvalsan, let's add exporters in the first round and merge. Then let's have another PR so the rest won't block exporters.

**Comment by shyamvalsan** [comment]
@ilyam8 I made another push. please take a look. (If there's still issues, feel free to jump in and update)

I have only taken out 2-3 entries where the link pointed to something with no useful info about the exporter. The others are linking to the exporter info even if it is not in github. So I am keeping them for now. 

cc: @sashwathn @Ancairon

**Review by sashwathn — APPROVED** [review]

**Pull request merged**